docs/user_guide/sqlserver/collection_scripts.md

```diff
@@ -155,15 +157,15 @@ In order to begin running the Database Migration Assessment Collection process,
   The following parameters can be specified:
   - -operation \*\* Required (create, start, stop, delete, collect, createemptyfile, help)
   - -instanceType \*\* Required (default, managed)
-  - -mssqlInstanceName \*\* Required if instanceType is "managed" (should be the instance name without the server name)
+  - -managedInstanceName \*\* Required if instanceType is "managed" (should be the instance name without the server name)
 
 To create and start the perfmon collection:
 
         For a default instance:
             ManageSqlServerPerfmonDatset.bat -operation create -instanceType default
 
         For a named instance:
```

Comment:
Should this be "For a managed instance:" ?

Reply:
The documentation and scripts have been updated to make the instance type either "default" or "named" and when named the new parameter that needs to be provided is "-namedInstanceName"